switchboard/dds_util.py

```diff
@@ -276,6 +276,10 @@ def get_action_template_text(self, action_name):
             raise RuntimeError('No email template found')
 
     def get_delivery(self):
```

Comment:
I actually don't see anywhere in the code that calls a `DDSUtil.get_delivery()`, so I don't know how this will help.

Reply:
This is a change to `DeliveryDetails` not `DDSUtil` but it doesn't look like that gets called either.